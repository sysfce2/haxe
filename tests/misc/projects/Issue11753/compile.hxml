-main Main2
--hl bin/main.hl
-D message.reporting=pretty
-D message.no-color
