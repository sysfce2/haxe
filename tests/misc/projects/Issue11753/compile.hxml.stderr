[WARNING] Main2.hx:17: characters 8-11

 17 |  $type(foo); // Unknown<0>
    |        ^^^
    | Unknown<0>

[WARNING] Main2.hx:19: characters 8-11

 19 |  $type(foo); // Unknown<0> : { doWithBar : () -> Unknown<1> }
    |        ^^^
    | Unknown<0> : { doWithBar : () -> Unknown<1> }

[WARNING] Main2.hx:20: characters 8-21

 20 |  $type(foo.doWithBar); // () -> Unknown<0>
    |        ^^^^^^^^^^^^^
    | () -> Unknown<0>

[WARNING] Main2.hx:23: characters 8-11

 23 |  $type(foo); // Null<{ doWithBar : () -> Unknown<0> }>
    |        ^^^
    | Null<Unknown<0> : { doWithBar : () -> Unknown<1> }>

[WARNING] Main2.hx:24: characters 8-21

 24 |  $type(foo.doWithBar); // () -> Unknown<0>
    |        ^^^^^^^^^^^^^
    | () -> Unknown<0>